plugins/workbench-resources/src/components/Workbench.svelte

```diff
@@ -184,11 +187,14 @@
   async function updateWindowTitle (loc: Location) {
     const ws = loc.path[1]
     const docTitle = await getWindowTitle(loc)
+    const unreadCountTitle = newNotificationsCount > 0 ? '(' + newNotificationsCount + ') ' : ''
     if (docTitle !== undefined && docTitle !== '') {
```

Comment:
Given that there are various possible titles it might be confusing for some of them. E.g. `(3) UBER-11 - Platform` is misleading as there's no clue that (3) refers to the unread's count. 
<img width="372" alt="image" src="https://github.com/hcengineering/anticrm/assets/20221821/7520220c-8cf8-4714-b551-9a650fe1dfd4">

Reply:
Added a label for it
![image](https://github.com/hcengineering/anticrm/assets/37306501/9f443405-58ce-4b3d-b507-ba462817aa26)

Reply:
Let's check with the UX team whether it works. If someone doesn't mark notifications as read he'd always see something like `Inbox (99+) - ...` which in case there are a lot of other tabs opened might be the only thing visible from the whole tab title.
Another, yet similar case if you have several tasks opened (and a lot of other tabs) you'd see:
`TSK-13 - WS` | `TSK-22 - WS` | `TSK-33-WS` and once a notification comes you'd see:
`Inbox (1) - TS` | `Inbox (1) - TS` | `Inbox (1) - TS`
So maybe this counter should be placed differently.